src/aiida/cmdline/commands/cmd_profile.py

```diff
@@ -136,17 +136,29 @@ def profile_setup():
 @setup.SETUP_BROKER_HOST()
 @setup.SETUP_BROKER_PORT()
 @setup.SETUP_BROKER_VIRTUAL_HOST()
-@options.NON_INTERACTIVE()
+@options.INTERACTIVE()
 @click.pass_context
 def profile_configure_rabbitmq(ctx, profile, **kwargs):
     """Configure RabbitMQ for a profile.
 
     Enable RabbitMQ for a profile that was created without a broker, or reconfigure existing connection details.
     """
+    from aiida.brokers.rabbitmq.defaults import detect_rabbitmq_config
+
     profile.set_process_controller(name='core.rabbitmq', config=kwargs)
     ctx.obj.config.update_profile(profile)
     ctx.obj.config.store()
 
+    broker_config = detect_rabbitmq_config(**kwargs)
+
+    if broker_config is None:
+        connection_params = {key: value for key, value in kwargs.items() if key.startswith('broker_')}
+        echo.echo_warning(
+            f'RabbitMQ configured but unable to connect to RabbitMQ server with configuration: {connection_params}'
+        )
+    else:
+        echo.echo_success('Successfully connected to RabbitMQ server.')
```

Comment:
I wonder if we should move this check before actually updating the config. I would then also update the messages as follows:
```suggestion
    if broker_config is None:
        connection_params = {key: value for key, value in kwargs.items() if key.startswith('broker_')}
        echo.echo_warning(
            f'Could not connect to RabbitMQ with the provided connection parameters.
        )
    else:
        echo.echo_success('Connected to RabbitMQ with the provided connection parameters')
```
Generally, don't think we want something like `Successfully` in a `echo_success` because it is a bit superfluous.

I would then also perhaps add a final confirmation message at the end of the command that confirms the profile config has  been updated:
```
echo.echo_success(f'RabbitMQ configuration for `{profile.name}` updated to: {connection_params}')
```